scene/main/viewport.cpp

```diff
@@ -3893,6 +3893,17 @@ bool Viewport::get_canvas_cull_mask_bit(uint32_t p_layer) const {
 	return (canvas_cull_mask & (1 << p_layer));
 }
 
+bool Viewport::is_visible_subviewport() const {
+	if (!is_sub_viewport()) {
+		return true;
+	}
+	if (!is_attached_in_viewport()) {
+		return false;
+	}
```

Comment:
```suggestion
```

The code below performs exactly the same check and the code below is necessary because of the cast. So this call to `is_attached_in_viewport()` can be removed.